OnlineStore/OnlineStore.Data/Contexts/ProductsOrdersContext.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using OnlineStore.Data.Contexts.Configuration;
+using OnlineStore.Data.Entities;
+using static Microsoft.EntityFrameworkCore.DbLoggerCategory.Database;
+
+namespace OnlineStore.Data.Contexts
+{
+    public class ProductsOrdersContext : IdentityDbContext<User, Role, Int32>
+    {
+        public DbSet<Сategory> Categories { get; set; }
+        public DbSet<OrderItem> OrderItems { get; set; }
+        public DbSet<Order> Orders { get; set; }
+        public DbSet<Product> Products { get; set; }
+
+        public ProductsOrdersContext(DbContextOptions<ProductsOrdersContext> options) : base(options)
+        { }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionBuilder)
+        {
+            base.OnConfiguring(optionBuilder);
+
+        }
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.ApplyConfiguration(new CategoryConfiguration());
+            modelBuilder.ApplyConfiguration(new OrderConfiguration());
+            modelBuilder.ApplyConfiguration(new OrderItemConfiguration());
```

Comment:
You can apply all configurations in one line without explicitly specifying each configurator